packages/netlify-cms-core/src/components/Editor/EditorScrollSync/ScrollSyncPane.js

```diff
@@ -29,14 +29,14 @@ export default class ScrollSyncPane extends Component {
   };
 
   componentDidMount() {
-    this.node = this.props.attachTo || ReactDOM.findDOMNode(this)
+    this.node = this.props.attachTo || ReactDOM.findDOMNode(this) // eslint-disable-line react/no-find-dom-node
```

Comment:
`findDOMNode` looks like a last resort here, with `attachTo` being used if at all possible. Therefore, I just told ESLint to ignore it. @Benaiah If you have a better fix, have at it!